docs/reference-1.4.0.html

```diff
@@ -878,7 +878,7 @@ <h4 id='map-other-methods'>Other Methods</h4>
 	<tr id='map-zoomanim'>
 		<td><code><b>zoomanim</b>
 		<td><code><a href='#zoomanimevent'>ZoomAnimEvent</a></code></td>
-		<td>Fired at least once per zoom animation. For continous zoom, like pinch zooming, fired once per frame during zoom.</td>
+		<td>Fired at least once per zoom animation. For continuous zoom, like pinch zooming, fired once per frame during zoom.</td>
```

Comment:
This file is generated automatically from code comments, can you please update corresponding comment?
It's right here:

https://github.com/Leaflet/Leaflet/blob/b2dd61d02fea3c5cb8c155737afe8a53ca25d400/src/map/Map.js#L1676-L1678